rclpy/src/rclpy/_rclpy.c

```diff
@@ -1693,6 +1693,69 @@ rclpy_remap_topic_name(PyObject * Py_UNUSED(self), PyObject * args)
   return result;
 }
 
+
+/// Expand and remap a topic name
+/**
+ * Raises ValueError if the capsule is not the correct type
+ *
+ * \param[in] pynode Capsule pointing to the node
+ * \param[in] topic_name topic string to be remapped
+ * \param[in] only_expand when `false`, remapping rules are ignored
+ * \param[in] is_service `true` for service names, `false` for topic names
+ * \return expanded and remapped topic name
+ */
+static PyObject *
+rclpy_resolve_name(PyObject * Py_UNUSED(self), PyObject * args)
+{
+  PyObject * pynode = NULL;
+  PyObject * pyname = NULL;
+  bool only_expand = false;
+  bool is_service = false;
```

Comment:
I think using `const char *` for string is correct, but maybe try using `int` for the bool values (just a shot in the dark).

Reply:
:bow: I think that was an accurate short in the dark.
I know "p" is expecting `int`, but I always forget it :grimacing: (and untyped varargs don't help).